CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
Agree with @dennyverbeeck. The reason Kafka has it, is because their repo is a clone of a SVN repository (see https://git.apache.org). In SVN, having a branch is the only way to properly manage a release, but for Git, the usual way is a tag.

Technically, SVN just stores directories and the semantics of trunk, branches and releases is managed by a standard directory structure. Git has native branches and tags.